### DISCUSSION TITLE
【2022.11.16】community meeting notes

### DISCUSSION THREAD

**Discussion by Stevent-fei** [sealerio/sealer#1871] [Community meeting]
hi all, 今晚8点，有咱们sealer的开发者会议，大家有什么议题可以提前准备哦。

**Comment by kakaZhou719** [comment]
1.官网doc 更新。 https://github.com/sealerio/sealer.cool/pull/18
2.sealer 集群运维侧命令支持 https://github.com/sealerio/sealer/issues/1861

**Comment by starComingup** [comment]
k3s installation introduce and k3s runtime module design discussion.

**Reply by cubxxw** [reply]
https://github.com/sealerio/sealer/issues/1859

**Comment by YTGhost** [comment]
Using Kyverno to support container runtime neutrally #1872

**Comment by zhy76** [comment]
e2e test in container

**Comment by kakaZhou719** [comment]
【会议纪要】
1. k3s,k0s分享。@starComingup,k0s代码对接新架构接近完成，需要社区支持conatinerd runtime. k3s由@cubxxw
同学继续调研中，@starComingup的comment作为主导。
2. sealer 支持containerd 的container runtime：@Stevent-fei,@kakazhou719 
3. 调研kyverno以及介绍kyverno的使用流程和show demo；后续需要确认 sealer如何结合kyverno，以及kyverno对k03,k3s的支持，包括 image policy的 map 策略@YTGhost
4. e2e代码已经运行成功，后续需要对接github action或者是使用通用的CI工具。@zhy76
5. rootfs 的规范由@VinceCui ，下周五之前提交 proposal。
6. 0.9.0 发布预计时间，需要汇总发布的前遗留事项@Starnop
7. sealer 官网文档需要有issue 作为todolist跟踪，@kakazhou719提供。
